### PR TITLE
Fix misalignment of merchandising slots on firefox

### PR DESCRIPTION
**Pull request by emma-imber**
## What does this change?

Adds a margin of zero to make sure that the browser doesn't add a margin around the merch slot content, causing misalignment.

Before:
Screenshot 2022-10-18 at 15 05 24

After:
Screenshot 2022-10-19 at 17 24 34

### CODE REVIEW

**Comment by emma-imber** [comment]
> Will this affect templates than merchandising?

Fair point - margin:0 is being automatically applied by chrome on most templates but it wouldn't hurt to make this css change more specific for now, to avoid edge cases

File: legacy/src/capi-multiple-hosted/web/index.scss [hunk 1]
```diff
@@ -28,3 +28,7 @@
     fill: #ffffff;
   }
 }
+
+body {
```
Comment (arelra) [inline legacy/src/capi-multiple-hosted/web/index.scss:32]: Entirely your call but is there scope to create a shared `_merchandising` CSS file from which any shared styling can be imported?

**Comment by domlander** [comment]
> > Will this affect templates than merchandising?
> 
> Fair point - margin:0 is being automatically applied by chrome on most templates but it wouldn't hurt to make this css change more specific for now, to avoid edge cases

Ah OK interesting. Maybe it's a good thing to set margin:0 generally if Chrome is doing that automatically, then deal with edge cases more specifically. Will leave it to your judgement.

**Review by domlander — APPROVED** [review]

File: legacy/src/capi-multiple-hosted/web/index.scss (hunk 1 above)
Comment (emma-imber) [reply legacy/src/capi-multiple-hosted/web/index.scss:32]: Yep that's probably a good shout! I'll update accordingly

**Pull request merged**